test/integration/test_remote_files.py

```diff
@@ -115,6 +118,43 @@ def test_fetch_from_import(self):
 
         assert os.path.exists(os.path.join(self.library_dir, "a"))
 
+    def test_fetch_from_drs(self, celery_worker):
+        # This test needs an additional worker because the md5 hashing task must run in parallel
```

Comment:
This must not be cleaning up correctly right... seems more likely than the coverage taking too long. Let me see if I have another idea for a work around.